projects/batfish/src/org/batfish/grammar/cisco/CiscoLexer.g4

```diff
@@ -7110,6 +7161,11 @@ M_AsPath_DEC
    F_Digit+ -> type ( DEC ) , popMode
 ;
 
+M_AsPath_In
```

Comment:
I don't understand. Does XXX stand for the same thing in the first and second line? If not, can you provide a sample definition for the thing the second XXX refers to?